src/common/snippets/include/snippets/pass/extract_reshapes_from_mha.hpp

```diff
@@ -29,10 +30,46 @@ namespace pass {
  *
  * @ingroup snippets
  */
-class ExtractReshapesFromMHA: public ov::pass::MatcherPass {
+class ExtractPairsAfterMatmul : public ov::pass::MatcherPass {
 public:
-    OPENVINO_MATCHER_PASS_RTTI("snippets::pass::ExtractReshapesFromMHA");
-    ExtractReshapesFromMHA();
+    OPENVINO_MATCHER_PASS_RTTI("snippets::pass::ExtractPairsAfterMatmul");
+    ExtractPairsAfterMatmul();
+};
+
+/**
+ * @interface RankUpgradeToRankReduction
+ * @brief This pass tries to transfer rank upgrade reshape from MHA branch to rank reduction reshape in MHA input branch
```

Comment:
As far as I understand, this transformation supports only static shapes. Could you please reflect it in the description?

Reply:
done, thanks Vladislav!